storage.rules

```diff
@@ -0,0 +1,8 @@
+rules_version = '2';
+service firebase.storage {
+  match /b/{bucket}/o {
+    match /users/{uid}/{allPaths=**} {
+        allow read, write, update: if request.auth.uid == uid
+    }
+  }
```

Comment:
Are we missing the rule for the venue?

Reply:
nope. storing the data in /users/{uid}/venues/{venuename}/...

There is no way of querying the firestore to ensure the user own the venueId so hypothetically any {venuename} could be used. This is not a security concern though as the venue in firestore stores the download url.